frontend/src/app/libs/event-info/services/event-info.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { Task, TasksTypes } from '../../landing/models/task.model';
+
+@Injectable({
+	providedIn: 'root'
+})
+export class EventInfoService {
+
+	constructor() { }
+
+	public fetchTasks(): Observable<Task[]> {
+		// temprorary return mock data
+		return of([{
```

Comment:
one more text type here is `text`, when user see the question and just text editor and submit button in task